[SimpleFSDP] update simplefsdp readme (#1328)
pytorch/torchtitan: torchtitan/experiments/simple_fsdp/README.md

[![integration tests](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_8gpu_simple_fsdp.yaml/badge.svg?branch=main)](https://github.com/pytorch/torchtitan/actions/workflows/integration_test_8gpu_simple_fsdp.yaml?query=branch%3Amain)
[![arXiv](https://img.shields.io/badge/arXiv-2411.00284-b31b1b.svg)](https://arxiv.org/abs/2411.00284)

> 💡 **Note**: SimpleFSDP's composability with mixed precision training and tensor parallel require updates from latest PyTorch. Users will need to install the latest PyTorch following [here](https://github.com/pytorch/torchtitan/blob/8850c5a3cfe59b24ea740999e0055cc77b67a229/.github/workflows/integration_test_8gpu_simple_fsdp.yaml#L43) to use SimpleFSDP.

Reviewer: maybe
```suggestion
💡 **Note**: SimpleFSDP's composability with Mixed Precision Training and Tensor Parallel requires updates from latest PyTorch , which can be installed via
```
```bash
pip3 install --pre torch --index-url https://download.pytorch.org/whl/nightly/cu126 --force-reinstall
```
